src/utils/configuration.ts

```diff
@@ -33,19 +33,39 @@ const getServerConfigurationValue = <T>(configName: keyof typeof configurationSc
 }
 
 const getPortNumber = (): number => {
-    return getServerConfigurationValue<number>('port', 8084)
+    return getServerConfigurationValue<number>('port', 7863)
 }
 
-const getPath = (): string => {
-    return getServerConfigurationValue<string>('binaryPath', '.bin/runme')
+const getPath = (basePath: string): string => {
+    const baseUri = Uri.parse(basePath)
+    const binaryPath = getServerConfigurationValue<string>('binaryPath', 'bin')
+
+    // relative path
+    if (
+      !binaryPath.startsWith('/') &&
+      binaryPath.length > 2 &&
+      binaryPath[1] !== ':'
```

Comment:
I am unconvinced this relative check is going to work in many cases on windows. For instance, what if the user provides a url with `\` separators, does the drive thing (`D:\path\to\runme`), etc.

Probably you'd want to use something like `path.isRelative(binaryPath)`. Looked into it and I didn't see anything in the `Uri` API that could do this so you may have to use node's API.

Reply:
My two cents, Let's remove this setting entirely and prevent future issues.